opte/src/ddi/sync.rs

```diff
@@ -208,8 +208,8 @@ pub struct KRwLock<T> {
 
 #[cfg(all(not(feature = "std"), not(test)))]
 pub enum KRwLockType {
-    Driver = krw_type_t::RW_DRIVER as isize,
-    Default = krw_type_t::RW_DEFAULT as isize,
+    Driver = krw_type_t::RW_DRIVER.0 as isize,
```

Comment:
You'll want someone else to review these changes to sync.rs as it is not clear to me exactly what's going on.

Reply:
This LGTM. Just looks like the FFI enum definition now has a bit of extra type wrapping that needs to be unwrapped for the corresponding more canonical Rust enum?